start.sh

```diff
@@ -393,19 +477,19 @@ fi
 if [ $TESTNET_ENV != true ]
 then
     COMPOSE_FILES="${COMPOSE_FILES} -f docker-compose.local-frequency.yaml"
-    PROFILES="${PROFILES} --profile local-node"
+    PROFILES="--profile local-node"
```

Comment:
This should be initialized to an empty string on line 383 (old)/465 (new) above; some shells don't like if you reference a variable that hasn't been defined.